### PR TITLE
Add computation models with Tensorflow and loader/executor in c++

### PR DESCRIPTION
**Pull request by cassand**

### CODE REVIEW

File: powerspikegg/computation-models/match/Readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Create a model in python and execute in C++ 
+ 
+## Generating the model 
+ 
+To generate the model execute the script graph.py, this will export the trained model in the directory
+ log 
+ 
+## Export the model 
+ 
+Now we have to inject the variables (neural network weight) as statics nodes in the graph to do that
```
Comment (ArchangelX360) [inline powerspikegg/computation-models/match/Readme.md]: static*

into the graph (to inject into)

File: powerspikegg/computation-models/match/graph.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+  with tf.name_scope('hidden2'):
+    weights = tf.Variable(
+        tf.truncated_normal([hidden1_units, hidden2_units],
+                            stddev=1.0 / math.sqrt(float(hidden1_units))),
+        name='weights')
+    biases = tf.Variable(tf.zeros([hidden2_units]),
+                         name='biases')
+    hidden2 = tf.nn.relu(tf.matmul(hidden1, weights) + biases)
+  # Linear
+  with tf.name_scope('softmax_linear'):
```
Comment (ArchangelX360) [inline powerspikegg/computation-models/match/graph.py]: no softmax, change label

File: powerspikegg/computation-models/simple_py/src/simple.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+    saver = tf.train.Saver()
+
+    with tf.Session() as sess:
+        sess.run(init_op)
+        print (sess.run(serve, {x: [[1.0,1.0]]}))
+
+        print(saver.save(sess, "models/graph.chkp", write_meta_graph=True)) 
+
+
```
Comment (ArchangelX360) [inline powerspikegg/computation-models/simple_py/src/simple.py:22]: remove blank lines

File: powerspikegg/computation-models/simple_py/src/simple.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    # Add an op to initialize the variables.
+    init_op = tf.global_variables_initializer()
+
+    saver = tf.train.Saver()
+
+    with tf.Session() as sess:
+        sess.run(init_op)
+        print (sess.run(serve, {x: [[1.0,1.0]]}))
+
```
Comment (ArchangelX360) [inline powerspikegg/computation-models/simple_py/src/simple.py:19]: remove this blank line

File: powerspikegg/computation-models/utils/src/graph.cc [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#include "graph.h"
+
+namespace computation {
+namespace utils {
+
+tensorflow::Node* FindNodeWithNameInGraph(tensorflow::Graph* graph,
+                                          std::string name) {
+    for (tensorflow::Node* node : graph->nodes()) {
+        if (node->name() == name) return node;
```
Comment (ArchangelX360) [inline powerspikegg/computation-models/utils/src/graph.cc:9]: We should save a node label map (label -> node) to avoid this worst O(n) case in later PRs.

**Review by ArchangelX360 — CHANGES_REQUESTED** [review]

**Review by ArchangelX360 — APPROVED** [review]
LGTM.
